### ISSUE TITLE
how do I use arrow keys?

### ISSUE THREAD

**Issue by foice** [lambdalisue/jupyter-vim-binding#141]
I have managed to use the `.jupyter/custom/custom.js` file to put in some of the snippets kindly provided, as they suited me. However I want to have the possibility to move from one cell to the adjacent using the arrow keys, instead of god gived jk, is there any way to use arrows?

**Comment by lambdalisue** [comment]
Probably you need to map [`jupyter-notebook:select-previous-cell/jupyter-notebook:select-next-cell`](https://github.com/lambdalisue/jupyter-vim-binding/blob/master/lib/jupyter/shortcuts.js#L41-L42) to arrow keys as Jupyter's mappings rather than CodeMirror's.
Easiest way is using Keybaord Shortcut Editor of Jupyter which is explained in https://github.com/lambdalisue/jupyter-vim-binding#customization

**Comment by foice** [comment]
Yes, you are right, all the standard cusotmization from help menu of
jupyter still works to affect the Jupyter mode bindings, I though now one
had to go vim all the way!
